### PR TITLE
Fix shouldDisableNextStep in Wizard Statefull component

### PR DESCRIPTION
**Pull request by guillaumevincent**
shouldDisableNextStep props is not working on Wizard component.
This pull request is here to fix the bug.

### CODE REVIEW

File: packages/patternfly-react/src/components/Wizard/Patterns/StatefulWizardPattern.js [hunk 1]
```diff
@@ -42,8 +42,8 @@ class StatefulWizardPattern extends React.Component {
     // These functions are for StatefulWizardPattern only and should not be used in WizardPattern despite being passed down here.
     return (
       <WizardPattern
-        nextStepDisabled={shouldDisableNextStep(activeStepIndex)}
         {...otherProps}
+        nextStepDisabled={shouldDisableNextStep(activeStepIndex)}
```
Comment (jeff-phillips-18) [inline packages/patternfly-react/src/components/Wizard/Patterns/StatefulWizardPattern.js]: Shouldn't have to override the `otherProps` here. I believe the real issue is the `nextStepDisabled` field being in the props for the StatefulWizardPattern. Adding `nextStepDisabled` to the excludeKeys from the WizardPattern.propTypes and WizardPattern.defaultProps would be a better fix.

**Comment by jeff-phillips-18** [comment]
Please use `yarn commit` to correctly format the commit message for semantic release.

File: packages/patternfly-react/src/components/Wizard/Patterns/StatefulWizardPattern.js (hunk 1 above)
Comment (guillaumevincent) [reply packages/patternfly-react/src/components/Wizard/Patterns/StatefulWizardPattern.js]: ack let me do this

**Comment by mturley** [comment]
Thanks so much for fixing this @guillaumevincent ! I had a suspicion that it maybe didn't work, and I ran out of time and didn't need it in my first usage. It's been on my list for a while so I appreciate you taking care of it!

**Pull request merged**